constructor/construct.py

```diff
@@ -1079,68 +133,49 @@ def parse(path, platform):
     return res
 
 
-def verify(info):
-    types_key = {}  # maps key to types
-    required_keys = set()
-    obsolete_keys = set()
-    for key, required, types, descr in KEYS:
-        types_key[key] = types
-        if required:
-            required_keys.add(key)
-        if "Obsolete" in descr:
```

Comment:
This has been replaced by `deprecated=True` annotations plus the `DeprecatedFieldWarning(ValidationError)` subclass.